golang-external-secrets/values.yaml

```diff
@@ -1,7 +1,23 @@
 ---
+# Eventually we should aim to move these two under the golangExternalSecrets key
 mountPath: "hub"
 mountRole: "hub-role"
 
+golangExternalSecrets:
+  # This controls how ESO connects to vault
+  caProvider:
+    enabled: true # If vault is exposed via a route that is signed by a non internal CA you might want to disable this
+    hubCluster:
```

Comment:
Can we avoid the hub/nonhub terminology please?  The world shouldn't break, and should still make sense, if someone doesn't start with a clustergroup called "hub".

What about `vaultHostCluster`, and `vaultClientCluster` which better describe the roles here?